src/responder/kcm/kcm.c

```diff
@@ -140,6 +140,7 @@ static int kcm_get_config(struct kcm_ctx *kctx)
         goto done;
     }
 
+#ifndef HAVE_KCM_RENEWAL
```

Comment:
This will disable idle timeout during build time. Wouldn't it be better to disable it only if renewals are enabled in configuration?

Reply:
You are correct, new version implements this behaviour.